client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionSection/InteractionDetailsFields/InteractionTimeForm.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import {Controller, useFormContext} from 'react-hook-form';
+import {isValid} from 'date-fns';
+import {Checkbox, FormControlLabel, Grid} from '@material-ui/core';
+import FormInput from 'commons/FormInput/FormInput';
```

Comment:
order of imports

Reply:
Hopefully fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/commit/bd9413139437352056d0aaff99583d726152f8a1)